web/models/user.ex

```diff
@@ -78,11 +81,22 @@ defmodule CodeCorps.User do
   def update_changeset(struct, params) do
     struct
     |> changeset(params)
-    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website, :base64_photo_data])
+    |> cast(params, [:first_name, :last_name, :twitter, :biography, :website, :base64_photo_data, :state_transition])
```

Comment:
The check seems redundant to me, but I went ahead and added it. What's your reasoning behind it? If someone gets the transition name wrong, won't they go to the transition file?